x/gov/src/client/tx_handler.rs

```diff
@@ -35,23 +40,26 @@ impl TxHandler for GovClientHandler {
                 proposal_id,
                 depositor: from_address,
                 amount,
-            })),
+            })
+            .into()),
```

Comment:
May you collect this into variable and do `into` in one place?

Reply:
Thanks, it's better solution.